src/pages/AssetDetailsPage.vue

```diff
@@ -24,29 +25,22 @@ const { width } = useWindowSize();
 
 const hashType = computed(() => (width.value < HASH_BREAKPOINT ? 'medium' : 'full'));
 
-const assetName = computed(() => {
+const assetDefinitionId = computed(() => {
   const name = router.currentRoute.value.params['id'];
+  const rest = router.currentRoute.value.hash;
 
-  invariant(typeof name === 'string', 'Expected string');
+  const assetId = AssetIdSchema.parse(name + rest);
```

Comment:
Why parsing `:id` in the route as `AssetId`? It is supposed to be `AssetDefinitionId`, e.g. `roses#wonderland`.